docs/preset-env.md

```diff
@@ -229,38 +229,64 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-> This option adds direct references to the `core-js` module as bare imports. Thus `core-js` will be resolved relative to the file itself and needs to be accessible. You may need to specify `core-js@2` as a top level dependency in your application if there isn't a `core-js` dependency or there are multiple versions.
+> When using this options, you also need to configure the [`corejs`](#corejs) option.
 
 This option configures how `@babel/preset-env` handles polyfills.
 
+This option adds direct references to the `core-js` module as bare imports. For this reason, you need to install `core-js@3` or `core-js@2` (depending on the value of the [`corejs`](#corejs) option) as a dependency of your application:
```

Comment:
```suggestion
When either the `usage` or `entry` options are used, `@babel-preset-env` will add direct references to `core-js` modules as bare imports (or requires). This means `core-js` will be resolved relative to the file itself and needs to be accessible.

Since `@babel/polyfill` was deprecated in 7.4.0, we recommend directly adding `core-js` and (if necessary), setting the version via the [`corejs`](#corejs) option.
```